trin-core/src/portalnet/overlay.rs

```diff
@@ -137,16 +136,22 @@ impl OverlayProtocol {
     pub async fn process_one_request(
         &self,
         talk_request: &TalkRequest,
-    ) -> Result<Response, OverlayRequestError> {
+    ) -> anyhow::Result<Response> {
```

Comment:
What do you think of putting at the top of the file:`type Result = anyhow::Result` so that all of these lines can just be: 
```suggestion
    ) -> Result<Response> {
```

Might improve readability a bit.

Reply:
I'm not so keen on this. Especially if we settle on two different error libraries. Yeah, it's a definite improvement in readability, but...
1 - it might confuse people new to codebase, familiar with Rust but not `anyhow`
2 - keeping track of which error handling library is used where in the codebase, I'd get frustrated with having to scroll to the top of each file to see if this import is there or not.